src/scripts/prepare-prod.js

```diff
@@ -12,21 +15,16 @@ const hostingStrategy = process.env.HOSTING_STRATEGY || "custom";
       )}\n`
     );
 
-    // Copying the api folder to the root directory
+    // Copying the vercel folder content to the root directory
     execa(
-      "cp",
-      ["-r", "node_modules/rasengan/lib/esm/server/functions/vercel/api", "."],
-      {
-        stdio: "inherit",
-      }
-    );
-
-    // Copying the vercel.json file to the root directory
-    execa(
-      "cp",
-      ["node_modules/rasengan/lib/esm/server/functions/vercel/vercel.json", "."],
+      copyCommand,
+      generateCopyExecaArray(
+        "node_modules/rasengan/lib/esm/server/functions/vercel",
+        "."
+      ),
       {
         stdio: "inherit",
+        shell: true,
```

Comment:
What this option does when enabled ?

Reply:
the shell option is required for Windows Environment

Reply:
Okay, well received.